src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoLineWrapCheckTest.java

```diff
@@ -35,14 +35,14 @@
     @Override
     protected String getPath(String filename) throws IOException {
         return super.getPath("checks" + File.separator
-                + "whitespace" + File.separator + filename);
+                + "whitespace" + File.separator  + "nolinewrap" + File.separator + filename);
     }
 
     @Test
     public void testCaseWithoutLineWrapping() throws Exception {
         final DefaultConfiguration checkConfig = createCheckConfig(NoLineWrapCheck.class);
         final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
-        verify(checkConfig, getPath("InputNoLineWrapGood.java"), expected);
+        verify(checkConfig, getPath("InputnolinewrapGood.java"), expected);
```

Comment:
> that name was already being used 

We will eventually fix that. As long as file is only one in directory it is fine for now.
MEZk is correct in that file name should be camel case like check name.